OWML.ModHelper.Menus/ModInputCombinationElement.cs

```diff
@@ -0,0 +1,171 @@
+﻿using OWML.Common;
+using OWML.Common.Menus;
+using OWML.ModHelper.Input;
+using OWML.ModHelper.Events;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+namespace OWML.ModHelper.Menus
+{
+    public class ModInputCombinationElement : ModToggleInput, IModInputCombinationElement
+    {
+        private const float ScaleDown = 0.75f;
+
+        public ILayoutManager Layout { get; private set; }
+
+        override public string Title
+        {
+            get => _combination;
+            set
+            {
+                _combination = value;
+                UpdateContents();
+            }
+        }
+
+        private string _combination;
+        private readonly GameObject _layoutObject;
+        private readonly IModInputHandler _inputHandler;
+
+        private static IModInputCombinationElementMenu _popupMenu;
+
+        public ModInputCombinationElement(TwoButtonToggleElement toggle, IModMenu menu, IModInputCombinationElementMenu popupMenu, IModInputHandler inputHandler, string combination = "") : base(toggle, menu)
```

Comment:
Can some of the contents be moved to Initialize?

Reply:
I made separate method "SetupButtons"